src/vcDBF.cpp

```diff
@@ -719,7 +730,7 @@ udResult vcDBF_RemoveField(vcDBF *pDBF, uint16_t fieldIndex)
   return result;
 }
 
-udResult vcDBF_GetRecord(vcDBF *pDBF, vcDBF_Record **ppRecord, uint32_t recordIndex)
+udResult vcDBF_GetRecord(vcDBF *pDBF, vcDBF_Record **ppRecord, uint16_t recordIndex)
```

Comment:
You changed this from `uint32_t`, you should have reverted it back to `uint32_t`.

Reply:
fixed.